agent/connectionchecker/connection_checker_test.go

```diff
@@ -99,9 +99,6 @@ func TestConnectionChecker(t *testing.T) {
 				Type:    inventorypb.ServiceType_MONGODB_SERVICE,
 				Timeout: durationpb.New(3 * time.Second),
 			},
-			expectedErr: `\(Unauthorized\) (?:command getDiagnosticData requires authentication|` +
```

Comment:
can we check if it's `mongod` service and based on that try to run getDiagnosticData to make sure that we can get this data?